aea/aea_builder.py

```diff
@@ -433,6 +434,8 @@ def set_decision_maker_handler(
                     dotted_path, class_name, file_path, e
                 )
             )
+            raise  # log and reraise cause looks we should not build an agent from invalid configuraion
```

Comment:
`log and re-raise because we should not build an agent from an. invalid configuration`